specs/multispec.yml

```diff
@@ -0,0 +1,21 @@
+version: 1
+
+specs:
+  distroinfo:
+    fedora:
+      distros:
+        - fedora-26-x86_64
+      vendor: "Fedora Project"
+  variant:
+    upstream:
+      image_name: modularitycontainers/memcached
+      readme_description: |-
+        [![Build Status](https://travis-ci.org/container-images/memcached.svg?branch=master)](https://travis-ci.org/container-images/memcached)
+
+        There is a detailed information about this image available in its [help
+        page](https://github.com/container-images/memcached/blob/master/help/help.md).
+    downstream:
+      image_name: f26/memcached
```

Comment:
this is not correct:
1. registry is missing
2. f26 should probably come from the spec